src/reformulate.jl

```diff
@@ -1,82 +1,387 @@
+################################################################################
+#                              LOGICAL VARIABLES
+################################################################################
 """
-    reformulate_disjunction(m::Model, disj...; bin_var, reformulation, param)
 
-Reformulate disjunction on a JuMP model.
+Create binary (indicator) variables for logic variables.
+"""
+function _reformulate_logical_variables(model::JuMP.Model)
+    for (lv_idx, lv_data) in _logical_variables(model)
+        var = JuMP.ScalarVariable(_variable_info(binary=true))
+        bvref = JuMP.add_variable(model, var, lv_data.name)
+        _indicator_to_binary(model)[lv_idx] = JuMP.index(bvref)
+    end
+end
```

Comment:
TODO: add start value from the logical variable

Reply:
done